app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,69 @@
+<div id="driver-detail-page">
+
+  <div class="summary-container">
+    <section class="summary">
+      <section class="pic">
+          <%= image_tag "http://placekitten.com/50/50", alt: "profile pic" %>
+      </section>
+      <section class="namenumreq">
+
+        <span class="name"><%= @driver.name %></span>
+
+
+        <span class="num"><%= @driver.vin %></span>
+
+
+        <span class="totalspent">Total Earnings: $<%= @driver.total_earnings%></span>
+
+          Average Rating: <%= @driver.average_rating%>
+
+      </section>
+      <section class="totalmods">
+
+          <%= link_to "Edit", edit_driver_path(@driver), class: "button" %>
+
+
+          <%= link_to "Delete", driver_path(@driver), method: :delete, data: {confirm: "Are you sure you want to delete the driver #{@driver.name}?"}, class: "button" %>
+      </section>
+
+    </section>
+  </div>
+
+    <section class="passenger-trips">
+
+      <h2 class="yourtrips">Driver Trips</h2>
+    <section class="trips-container">
+      <ul class="trips-headers">
+        <li class="tripsid">ID</li>
+        <li class="tripsdate">Date</li>
+        <li class="tripsdriver">Driver</li>
+        <li class="tripspassenger">Passenger</li>
+        <li class="tripsprice">Price</li>
+        <li class="tripsrating">Rating</li>
+        <li class="tripsdelete">Delete</li>
+      </ul>
+
+      <% @driver.trips.each do |trip| %>
+      <ul class="tripsentries">
+        <li class="tripsid"><%= link_to trip.id, trip_path(trip.id) %></li>
+        <li class="tripsdate"><%=trip.date%></li>
+        <li class=
+        "tripsdriver"><%= link_to trip.driver.name, driver_path(trip.driver.id) %></li>
+        <li class="tripspassenger"><%= link_to trip.passenger.name, passenger_path(trip.passenger.id) %></li>
+        <li class="tripsprice">$<%=trip.cost%></li>
```

Comment:
Small comment, but this view doesn't format the cost to have a decimal two places in ;) trips costs are in the thousands haha